coalib/bears/Bear.py

```diff
@@ -273,6 +278,175 @@ def log_message(self, log_message, timestamp=None, **kwargs):
     def run(self, *args, dependency_results=None, **kwargs):
         raise NotImplementedError
 
+    def parenthesis_split(self, sentence, separator=',',
+                          lparen='(', rparen=')'):
+        nb_brackets = 0
+        sentence = sentence.strip(separator)
+        final = [0]
+        try:
+            for i, c in enumerate(sentence):
+                if c == lparen:
+                    nb_brackets += 1
+                elif c == rparen:
+                    nb_brackets -= 1
+                elif c == separator and nb_brackets == 0:
+                    final.append(i)
+                if nb_brackets < 0:
+                    raise ValueError('Invalid arguments to --profile-bears')
+            final.append(len(sentence))
+            if nb_brackets > 0:
+                raise ValueError('Invalid arguments to --profile-bears')
+        except ValueError as err:
+            self.err(err.args[0])
+            return
+        return([sentence[i:j].strip(separator) for i, j in zip(final, final[1:])])
+
+    def pstats_config(self, prof, stream, profile_bears):
+        flag = 0
```

Comment:
Line is longer than allowed. (81 > 80)

*Origin: LineLengthBear, Section: `linelength`.*